.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+## Required information
+
+**Operating system:**
+Output of: `uname -a`
```

Comment:
Maybe add that this is for Linux ... maybe also other Unix. Windows might have something similar